### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by iqtedar123** [hemang249/flutter_tesla_auth#1]
Hi,
First of all impressive work.
Secondly, does it still work?
I had something similar but I get an error when exchanging the token:
`invalid code_verifier`.
Thanks.

**Comment by KartikAiyer** [comment]
Yeah I'm curious to know if this works too. Looking at some of the auth code specifically the first POST request, the headers seem to be different than whats in the unofficial Tesal API docs. I don't see the hidden fields being propagated. @iqtedar123 did you try running this app ? I only have dart 3 and the null safety issues are stopping me from building the app.

**Comment by iqtedar123** [comment]
> Yeah I'm curious to know if this works too. Looking at some of the auth code specifically the first POST request, the headers seem to be different than whats in the unofficial Tesal API docs. I don't see the hidden fields being propagated. @iqtedar123 did you try running this app ? I only have dart 3 and the null safety issues are stopping me from building the app.

I got it working by making some changes. The main functionality is there. You just need to add breakpoints everywhere to figure it out.
